### DISCUSSION TITLE
posts/2021/d8c3e71b/

### DISCUSSION THREAD

**Discussion by giscus** [GZTimeWalker/blog-comments#9] [Announcements]
# posts/2021/d8c3e71b/

又是一年，年至弱冠。

https://blog.gztime.cc/posts/2021/d8c3e71b/

**Comment by swapfloor** [comment]
看到优秀的同学，我会很崇拜，不像我，打比赛**啥也不会**，绩点烂的**爆炸**。每天就看看别人博客。感觉唯一有趣的事就是自己*写写代码*。感觉你非常非常厉害！！，非常崇拜。

**Comment by amy-77** [comment]
正经人
